plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -23,4 +23,20 @@
     <persistent value="true" />
   </extension>
   
+  <extension 
+      name="App Engine Descriptor Validator" 
+      point="org.eclipse.wst.validation.validatorV2" 
+      id="com.google.cloud.tools.eclipse.appengine.validation.MyValidator">
```

Comment:
I'd prefer this also to be renamed. Could be `...AppEngineWebXmlValidator` like the class name.